django/cantusdb_project/main_app/urls.py

```diff
@@ -73,6 +73,8 @@
     ProvenanceAutocomplete,
     ProofreadByAutocomplete,
 )
+from django.conf.urls import handler400, handler404
```

Comment:
Are these used?

Reply:
I see you’ve removed them — great! If you’re not using a code editor that highlights these sort of syntax or style errors, you should. They are widely available and help catch these sorts of things before it needs “human linting”